galaxy/main/models.py

```diff
@@ -1153,3 +1191,35 @@ class Meta:
         null=True,
         validators=[MinValueValidator(0), MaxValueValidator(5)]
     )
+
+
+class ContentRule(BaseModel):
```

Comment:
It's also harder to review the rules and to get a consistent source of truth on what the rules are supposed to be if its in the database.

Reply:
@chouseknecht should also weigh in since there was talk about having the rule to severity editable in djangoadmin, but in thinking about testing and maintaining over time I would lean towards putting into constants.

Reply:
If we're editing the severity levels in django admin, wouldn't we need a way to recompute the quality score for all the content items that have the error messages that are being changed?

Reply:
updating a severity in django admin would not start an automated score recompute, so that seems like another advantage of having the rule to severity mapping as constants.

Reply:
@awcrosby 

Let's put in the DB for now, as it will help us iterate and test faster.

Please add a migration that sets our initial best-guess set of rules and scoring values. The migration will at least put some initial values in everyone's local dev environment.

Reply:
I agree with @newswangerd , I would prefer to see rules in the code or configuration file rather than in the database.